docs/custom_promise_types/gpg_keys_using_lib.py

```diff
@@ -0,0 +1,154 @@
+"""Custom Promise Type module for adding missing gpg keys
```

Comment:
The only reason why I called my modules `from_scratch` and `using_lib` was because there were 2 different ones. I think this one should be just `gpg.py`, and the promise type should be `gpg`.